### PR TITLE
Add benefit to using automated build.

### PR DESCRIPTION
**Pull request by charleswhchan**
I am not sure if I have written this in the best possible way, but I think it is something to consider when deciding to build image locally/on-premise vs directly on Docker Hub.

### CODE REVIEW

**Comment by LK4D4** [comment]
ping @thaJeztah @moxiegirl @SvenDowideit

File: docs/docker-hub/builds.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ image was built exactly as specified.
 your repository on the Docker Hub registry.
 * Because the process is automated, Automated Builds help to
 make sure that your repository is always up to date.
+* Save network bandwidth and time pushing local Docker image
```
Comment (thaJeztah) [inline docs/docker-hub/builds.md]: My take on this:

`You save network bandwidth and time, not having to push local Docker images to Docker Hub.`

**Comment by thaJeztah** [comment]
Thanks @charleswhchan. Yes, I think this is a very good reason to use automated builds.

I added a suggestion for slightly different wording

**Comment by charleswhchan** [comment]
Looks good. I will update the patch later on ...

File: docs/docker-hub/builds.md (hunk 1 above)
Comment (moxiegirl) [inline docs/docker-hub/builds.md]: The comma is badly placed.

`Not having to push local Docker images to Docker Hub saves you both network bandwidth and time.`

Comment (charleswhchan) [inline docs/docker-hub/builds.md]: @thaJeztah, @moxiegirl: thanks for the suggestions. I will commit the last one.

**Comment by charleswhchan** [comment]
Edits committed.

**Comment by thaJeztah** [comment]
Thanks!

LGTM, I'll merge, counting @moxiegirl's suggestion as the second LGTM

**Pull request merged**